components/service/glean/src/main/java/mozilla/components/service/glean/ping/PingMaker.kt

```diff
@@ -39,12 +39,17 @@ internal class PingMaker(
      * @param pingName The name of the ping
      * @return sequence number
      */
+    @Synchronized
```

Comment:
This makes sense to me, in theory. However, I'm struggling to figure out a case in which that would happen :\

Reply:
Yeah, ditto.  I'm grasping at straws, honestly.  Also, it introduces an interkitten in Taskcluster that I can't reproduce locally...

Reply:
@travis79 @mdboom .. any idea?

Reply:
Maybe if we we hit the events limit, went over by 1 or more events, and then immediately went to background.  All other paths through here seem pretty isolated/sync.   That's the only situation I can come up with that might have us two separate execution threads in this same bit of code near the same time.

Reply:
Yeah.  I can't imagine anything else.  But that seems really unlikely to explain the high number of the dupes found in chutten's analysis.  (Plus, I'm not sure any of the dupes involve the event ping).  I also don't think we're getting a ton of events from fenix, but it's probably worth checking to see if we have any event pings with around 500 events in them.